src/settings-ui/Settings.UI/Flyout/AppsListPage.xaml

```diff
@@ -11,19 +11,25 @@
     xmlns:viewmodels="using:Microsoft.PowerToys.Settings.UI.ViewModels"
     mc:Ignorable="d">
 
-    <Grid Background="{ThemeResource LayerOnAcrylicFillColorDefaultBrush}">
+    <Grid Background="{ThemeResource LayerOnAcrylicFillColorDefaultBrush}"
```

Comment:
Why removing this? This was by design (see Quick Actions flyout)

Reply:
We now also lose the Acrylic layering and the separator line?

<img width="619" alt="image" src="https://user-images.githubusercontent.com/9866362/226325356-971f1822-4f60-4b0a-96f0-cfd5e9ea24a3.png">

Reply:
For the moment, indeed, but I can review this...

Reply:
Pretty much reverted. See latest comment.